.github/workflows/tox-pytest.yml

```diff
@@ -49,49 +139,78 @@ jobs:
         uses: actions/cache@v3
         id: cache-zenodo-datastore
         with:
-          path: ~/pudl-work/data/
+          path: ${{ env.PUDL_INPUT }}
           key: zenodo-datastore-${{ hashFiles('datastore-dois.txt') }}
-          restore-keys: |
-            zenodo-datastore-
 
-      - name: Log pre-test Zenodo datastore contents
-        run: |
-          mkdir -p ~/pudl-work/data/
-          find ~/pudl-work/data/
+      - name: Make cache/output dirs
+        run: mkdir -p ${{ env.PUDL_OUTPUT }} ${{ env.PUDL_INPUT}}
 
-      - name: Log SQLite3 version
-        run: |
-          which sqlite3
-          sqlite3 --version
+      - name: List workspace contents
+        run: find ~/pudl-work
```

Comment:
Should we hard code this to point at `/home/runner` rather than tilde as with the `PUDL_INPUT` values above?